ftl/BUILD

```diff
@@ -43,3 +47,35 @@ py_test(
         "@mock//:mock",
     ],
 )
+
+py_library(
+    name = "node_lib",
+    srcs = glob(["node/*.py"]),
+    deps = [":ftl_lib"],
+)
+
+# The node base image sets a default CMD, which causes issues if you run our image with no CMD.
+# So, reset it to [""] before building our py_image.
```

Comment:
Is this something we can/should upstream?

Reply:
I think we might actually want to not use this as our base, and instead start with distroless node or something else instead. I have to think about this a bit more.

Reply:
Ah, at first I thought this *was* the distroless base, but I see it clearly is not :)